packages/fields-auto-increment/src/test-fixtures.js

```diff
@@ -34,19 +38,32 @@ export const initItems = () => {
   ];
 };
 
-export const storedValues = () => [
-  { name: 'product1', orderNumber: 1 },
-  { name: 'product2', orderNumber: 2 },
-  { name: 'product3', orderNumber: 3 },
-  { name: 'product4', orderNumber: 4 },
-  { name: 'product5', orderNumber: 5 },
-  { name: 'product6', orderNumber: 6 },
-  { name: 'product7', orderNumber: 7 },
-];
+export const storedValues = matrixValue =>
```

Comment:
This feels like a strange name IMO since from reading the test, it's not actually the stored value, it's the value returned from graphql?